examples/v1beta1/trial-images/simple-pbt/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.9
```

Comment:
Can you add this Dockerfike to the [publish-trial-images.yaml](https://github.com/kubeflow/katib/blob/10f674a15512a2f1f2ebe52cab235d8702a2e1e4/.github/workflows/publish-trial-images.yaml)?